templates/100.html

```diff
@@ -16,9 +16,9 @@ <h1>100 theorems</h1>
 	<a href="https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository">"Editing files in another user's repository"</a>.
 	</p>
 
-    {% for theorem in hundred_theorems|selectattr('author') %}
+    {% for theorem in hundred_theorems|selectattr('formalized') %}
         <h5 class="markdown-heading mt-5" id="{{ theorem.id }}">{{ theorem.id }}. {{ theorem.title }} <a class="hover-link" href="#{{ theorem.id }}">#</a></h5>
-        <p>Author{% if ' and ' in theorem.author %}s{% endif %}: {{ theorem.author }}</p>
+        {% if theorem.author %}<p>Author{% if ' and ' in theorem.author %}s{% endif %}: {{ theorem.author }}</p>{% endif %}
```

Comment:
If there's no author data, I chose to just skip this line, but we could also say something like "Author data incomplete" or something like that to spur people into filling in the data.

Reply:
Good thinking. I think your version is fine.